static/sass/_utility_crop.scss

```diff
@@ -0,0 +1,31 @@
+@mixin blog-u-crop {
+  // Crop to 16:9
+  .u-crop--16-9 {
+    position: relative;
+
+    &::before {
+      content: "";
+      display: block;
+      padding-top: 56.25%;
```

Comment:
Magic number

Reply:
Puh, tbh I got all of this from blog.ubuntu.com. Will have to investigate